conf/eslint.json

```diff
@@ -88,10 +88,10 @@
         "no-return-assign": "off",
         "no-script-url": "off",
         "no-self-assign": "error",
-        "no-self-compare": "off",
+        "no-self-compare": "error",
```

Comment:
Ok, but my intent was to propose enabling them for everyone using `eslint:recommended`. :) Is it too late to change these and/or do you not agree they are fairly serious enough (or easy enough to workaround) so as to merit error messages (I could see the `blocked-scope-var` being a little more arguable than the others)?